src/dmd/statementsem.d

```diff
@@ -1825,6 +1811,30 @@ else
         result = s;
     }
 
+    private static extern(D) Expression applyDelegate(ForeachStatement fs, Expression flde,
```

Comment:
Please do not create functions with no Ddoc comment explaining the parameters and returns.

Reply:
These are private functions